test/relayclient/KnownRelaysManager.test.ts

```diff
@@ -390,4 +396,30 @@ contract('KnownRelaysManager 2', function (accounts) {
       assert.equal(sortedRelays[1][2].pctRelayFee, '4')
     })
   })
+
+  describe('#getAuditors()', function () {
+    const auditorsCount = 2
+    let knownRelaysManager: KnownRelaysManager
+    before(function () {
+      const activeRelays: RelayInfoUrl[] = [{ relayUrl: 'alice' }, { relayUrl: 'bob' }, { relayUrl: 'charlie' }]
```

Comment:
add preferred: ['alice', 'david' ]
to check box duplicate and non-duplicate urls..

Reply:
Not part of this PR